pkgs/os-specific/linux/swapspace/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, utillinux }:
+
+stdenv.mkDerivation rec {
+  pname = "swapspace";
+  version = "1.17";
+
+  src = fetchFromGitHub {
+    owner = "Tookmund";
+    repo = "Swapspace";
+    rev = "v${version}";
+    sha256 = "06xvmyy1fp94h00k9nn929j00ca3w12fiz07wf9az6srxa8i4ndz";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  patchPhase = ''
+    sed -e 's@"mkswap"@"${utillinux}/bin/mkswap"@' \
+      -e 's@"/sbin/swapon"@"${utillinux}/bin/swapon"@' \
+      -e 's@"/sbin/swapoff"@"${utillinux}/bin/swapoff"@' \
+      -i src/support.c src/swaps.c
```

Comment:
Please use substituteInPlace.

Reply:
@SuperSandro2000 that's a lot of extra work and I don't think it will be a problem, see https://github.com/NixOS/nixpkgs/pull/88093#discussion_r645970752

Reply:
It isn't really and sed fails silently. substituteInPlace at least prints a warning.

Reply:
it will take me at least 20 minutes to create the patch and figure out the correct incantation, and everytime the code changes, the patches have to change too...

Reply:
okay fine for me but please change patchPhase to postPatch to not break patches.